web/firefoxcom.js

```diff
@@ -384,6 +398,10 @@ class FirefoxExternalServices extends DefaultExternalServices {
     return new FirefoxPreferences();
   }
 
+  static setEditorStates(data) {
+    FirefoxCom.request("setEditorStates", data);
```

Comment:
For consistency with existing methods, in the `FirefoxExternalServices`-class, maybe use:
```suggestion
  static updateEditorStates(data) {
    FirefoxCom.request("updateEditorStates", data);
```